Python/ceval_macros.h

```diff
@@ -447,3 +447,8 @@ do { \
 #define EXIT_TO_TRACE() goto exit_to_trace
 #define EXIT_TO_TIER1() goto exit_to_tier1
 #define EXIT_TO_TIER1_DYNAMIC() goto exit_to_tier1_dynamic;
+
+static PyObject **common_constants[NUM_COMMON_CONSTANTS] = {
+    [CONSTANT_ASSERTIONERROR] = &PyExc_AssertionError,
```

Comment:
I had to store pointers-to-pointers because `PyExc_AssertionError` itself is not a constant and can't be used in a global.